scripts/build-json/build-json.js

```diff
@@ -39,4 +39,4 @@ function buildJSON(searchPaths) {
     return errors;
 }
 
-process.exit(buildJSON(process.argv.slice(2)));
\ No newline at end of file
+process.exitCode = buildJSON(process.argv.slice(2));
```

Comment:
I don't think this will work as-is, will it, because it's not in an async function? I don't know enough about Node to say whether my approach is definitely right but it seems to work (e.g. https://nodejs.dev/how-to-exit-from-a-nodejs-program talks about this a bit).